lms/static/scripts/frontend_apps/components/test/AppRoot-test.js

```diff
@@ -26,6 +48,10 @@ describe('AppRoot', () => {
   afterEach(() => {
     navigateTo(originalURL);
 
+    // Mounted AppRoot(s) must be unmounted after each test to prevent them
+    // from responding to URL changes in subsequent tests.
+    wrappers.forEach(w => w.unmount());
```

Comment:
Is this something we'll need to keep in mind for, in future, other components that are affected by routing?

Reply:
This component is affected because it contains `Switch` and `Route` components, which need to observe the current URL under the hood. Any others which do would also be affected.